src/main/java/com/gmail/nossr50/listeners/EntityListener.java

```diff
@@ -652,7 +659,14 @@ else if (livingEntity instanceof Tameable pet) {
      */
     @EventHandler(priority = EventPriority.LOWEST)
     public void onEntityDeathLowest(EntityDeathEvent event) {
-        mcMMO.getTransientMetadataTools().cleanLivingEntityMetadata(event.getEntity());
+        LivingEntity entity = event.getEntity();
+
+        // Clear metadata for Slimes/Magma Cubes after transformation events take place, otherwise small spawned slimes will not have any tags
+        if (Arrays.asList(EntityType.SLIME, EntityType.MAGMA_CUBE).contains(entity.getType())) {
```

Comment:
Make Arrays.asList(EntityType.SLIME, EntityType.MAGMA_CUBE) a static constant somewhere (could be this class or in somke util) and use that instead